cli/src/commands/create/bucket.rs

```diff
@@ -42,7 +40,7 @@ pub fn create(client: &Client, args: &CreateBucketArgs) -> Result<()> {
                 transform_tag: transform_tag.as_ref(),
             },
         )
-        .chain_err(|| ErrorKind::Client("Operation to create a bucket has failed".into()))?;
+        .context("Operation to create a bucket has failed")?;
```

Comment:
While we're touching error handling code, what do you think of making error messages start lowercase?

I think it makes sense because these are rarely proper sentences, and when chained with colons it looks more natural if they're lowercase.

```suggestion
        .context("operation to create a bucket has failed")?;
```

Reply:
I'm not convinced by the lowercase argument particularly - I'll leave it for another PR/bikeshed before manually touching all those lines.